### PR TITLE
Fix/patch aggregate rating

### LINKED ISSUE #90: Reviews generator fails patching 'aggregateRating'
When generating reservations with the feeder, the element passed to the function is wrong, leading into an error at every restaurant entity:

``` json
"aggregateRating": {
    "value": {
         "reviewCount" : 0,
         "ratingValue" : null
}
```

Also, we should add every `string` between single quotes inside the searching filters.

### PR DESCRIPTION
**Pull request by albertinisg**
This PR fixes #90 and updates the data image with the error fixed

### CODE REVIEW

File: server/utils.js [hunk 1]
```diff
@@ -765,10 +765,18 @@ function generateId(name, date) {
...
 
 function addConditionToQuery(listOfConditions, key, operator, value) {
+  var condition;
+
   if (!listOfConditions) {
     listOfConditions = [];
   }
-  var condition = key + operator + value;
+  if (typeof value === 'string' && key !== 'startTime') {
+    var quotedValue = '\'' + value + '\'';
```
Comment (jmcanterafonseca) [inline server/utils.js]: may you can avoid the scape if you use "'"

I withdraw this comment, it is clearer using the escape

File: server/utils.js [hunk 2]
```diff
@@ -765,10 +765,18 @@ function generateId(name, date) {
...
+    condition = key + operator + quotedValue;
+  } else {
+    condition = key + operator + value;
+  }
+
```
Comment (jmcanterafonseca) [inline server/utils.js:779]: probably better

```
var finalValue = value;
if (typeof value === 'string' && key !== 'startTime') {
   finalValue = '\'' + value + '\'';
}
condition = key + operator + finalValue;
```

**Comment by jmcanterafonseca** [comment]
LGTM with the suggestion fixed

thanks!

File: server/utils.js (hunk 2 above)
Comment (albertinisg) [inline server/utils.js:779]: True!

**Comment by jmcanterafonseca** [comment]
LGTM, thanks

**Pull request merged**